client/pages/tables.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import AppContext from '../lib/app-context';
+// import Redirect from '../components/redirect';
+import Navbar from '../components/navbar';
+import ListTables from '../components/listTables';
+
+export default class Tables extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tablesFetch: [],
+      isLoading: true,
+      isFreeModalSelected: false,
+      isAssignModalSelected: false,
+      tableSelected: null
+    };
+    this.freeTable = this.freeTable.bind(this);
+    this.assignTable = this.assignTable.bind(this);
+    this.noBtnClick = this.noBtnClick.bind(this);
+    this.yesBtnClick = this.yesBtnClick.bind(this);
+
+  }
+
+  componentDidMount() {
+    fetch('/api/get/tables')
+      .then(res => res.json())
+      .then(result => {
+        this.setState({ tablesFetch: result, isLoading: false });
+      });
+  }
+
+  freeTable(event) {
+    this.setState({ isFreeModalSelected: true, tableSelected: event.target.id });
+  }
```

Comment:
If you use `parseInt` when the table is selected, you won't have to `parseInt` when finding the correct table in your loop.